sdk/program/src/pubkey.rs

```diff
@@ -665,13 +665,15 @@ impl AsMut<[u8]> for Pubkey {
 
 impl fmt::Debug for Pubkey {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}", bs58::encode(self.0).into_string())
+        let _ = bs58::encode(self.0).into_string().fmt(f);
+        write!(f, "")
```

Comment:
is this some kind of "flush"?

Reply:
Ah. Good catch. We don't need this. It was removed in the recent commit.